clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/mapreduce/datawriter/reduce/VeniceReducer.java

```diff
@@ -134,23 +137,21 @@ private long getTotalIncomingDataSizeInBytes(JobConf jobConfig) {
     JobID jobID = null;
     RunningJob runningJob = null;
     Counters quotaCounters = null;
+
     try {
       hadoopJobClient = hadoopJobClientProvider.getJobClientFromConfig(jobConfig);
       jobIdProp = jobConfig.get(MAP_REDUCE_JOB_ID_PROP);
       jobID = JobID.forName(jobIdProp);
       runningJob = hadoopJobClient.getJob(jobID);
       quotaCounters = runningJob.getCounters();
       return MRJobCounterHelper.getTotalKeySize(quotaCounters) + MRJobCounterHelper.getTotalValueSize(quotaCounters);
-
     } catch (Exception e) {
-      /**
-       * Note that this will catch a NPE during tests unless the storage quota is set to
-       * {@link Store.UNLIMITED_STORAGE_QUOTA}, which seems quite messed up. It manifests as {@link runningJob}
-       * being null, which then prevents us from calling {@link RunningJob#getCounters()}. Obviously, this is
-       * not happening in prod, though it's not completely clear why...
-       *
-       * TODO: Fix this so that tests are more representative of prod
-       */
+      if (jobConfig.get("mapreduce.framework.name").equals("local")) {
```

Comment:
Can we replace these two literals with constants `MRConfig.FRAMEWORK_NAME`, `MRConfig.LOCAL_FRAMEWORK_NAME`?  Also is `0` correct return value?